src/server/__tests__/user.test.ts

```diff
@@ -67,48 +67,55 @@ describe('WHEN testing user endpoints', () => {
         });
 
         describe('WHEN changing password for the new user', () => {
-          const newPassword = 'newpass';
-          it('SHOULD return staus code 200', () => changePasswordRequest(newPassword, authToken).expect(200));
+          const password = 'newpass';
+          it('SHOULD return staus code 200', () => changePasswordRequest(password, authToken).expect(200));
 
           describe('WHEN password is changed with success', () => {
-            beforeEach(async () => (await changePasswordRequest(newPassword, authToken)).body);
+            beforeEach(async () => (await changePasswordRequest(password, authToken)).body);
 
             it('SHOULD be able to generate a new token and get "/me', async () => {
-              const newAuthToken = (await getTokenRequest({email: createUserPayload.email, password: newPassword})).body.access_token;
+              const newAuthToken = (await getTokenRequest({email: createUserPayload.email, password})).body.access_token;
               const me = (await getMeRequest(newAuthToken)).body;
               expect(me.email).toEqual(createUserPayload.email);
             });
           });
         });
 
-        describe('WHEN resetting password for the new user', () => {
-          it('SHOULD return staus code 200', () => resetPasswordRequest(createUserPayload.email).expect(200));
+        // TODO: Fix this. Maybe check that resetPasswordId is generated. In order to check that the resetPasswordId
+        // is generated we must return it as response. We can then also test /{userId}/changepassword
+        // describe('WHEN doing resetpassword for a user', () => {
```

Comment:
I agree. We should just have the check for status code 200. We should not have it return the reset token because it might be a vulnerability, if someone finds out the user id for that email aswell they can reset a user's password.